### ISSUE TITLE
[Feat] Scrubbing through a clip pauses the clip

### ISSUE THREAD

**Issue by lars-vc** [Rodeoclash/vodon-pro#57]
## Idea
Currently, when you scrub through a clip, the clip will be paused after clicking on the timeline. It would feel more natural that the video keeps playing if the video was playing before the scrub happens.
## Issues
If I understand correctly, the current implementation pauses because of performance limitations. I have also encountered issues with this, sometimes after scrubbing and quickly pressing play, the clip will play a second of where you were in the clip before jumping to the correct spot. Sometimes it doesn't even jump at all, this can be fixed by quickly pressing pause and play again, then it will jump.

**Comment by Rodeoclash** [comment]
You're correct in that this is a limitation of the system as it stands. After clicking to navigate on the timeline, we need to wait to ensure all videos are synced before we can continue playback.

**Comment by Rodeoclash** [comment]
I think it's worth doing a bit more investigation into the current system to see if I can buffer play / pause events to avoid needing to use the play / pause to get things moving again. I'll leave this ticket open until I've investigated.
